doc/requirements.txt

```diff
@@ -1,3 +1,4 @@
+argparse-formatter>=1.4
```

Comment:
These are requirements only needed for building the docs. The package requirements will already be met by installing signac, so this can be removed.